routes/fakes.go

```diff
@@ -72,31 +72,31 @@ func (s FakeInstanceStore) Destroy(instance models.Instance) error {
 }
 
 type FakeExecutor struct {
-	_CreateBtrfsSubvolume func(id int) error
-	_FinaliseImage        func(image models.Image) error
-	_CreateInstance       func(imageID int, instanceID int, port int) error
-	_DestroyImage         func(id int) error
-	_DestroyInstance      func(id int) error
+	_CreateBtrfsSubvolume func(ctx context.Context, id int) error
```

Comment:
We're not using this context in any tests atm - seems unnecessary to pass it down to the fake methods

Reply:
We're not but do you want your fake functions to deviate from the underlying implementation? I'd imagine that would get real confusing.

Reply:
I'd expect to add this at a point where we wanted to use these contexts in tests, but can understand the argument for consistency. Just trying to avoid adding any more noise to these tests!

Reply:
Yeah I think this is where writing custom fake structs for testing falls down slightly, compared to using for example testify's mocks. Not saying we should change it here but I like that everything is consistent with testify and the pattern requires very little thinking about.

Reckon custom fakes encourage many different ways of writing your interface and introspecting behaviour, which ideally you'd avoid in a test setup.

Just a comment, no need to action :)